## Problem Statement 1:
Blood glucose levels for obese patients have a mean of 100 with a standard deviation of 15. A researcher thinks that a diet high in raw cornstarch will have a positive effect on
blood glucose levels. A sample of 36 patients who have tried the raw cornstarch diet
have a mean glucose level of 108. Test the hypothesis that the raw cornstarch had an
effect or not

In [33]:
import math
sample_count = 36
sample_mean = 108
population_mean = 100
populationSigma = 15 #given that is standard deviation

lets consider null hypothesis  <br>
lets consider H0 = 100   <br>
and H1 is not = 100    <br>
lets assume alpha = 0.05  <br>




In [34]:
# lets calculate zScore value
zScore = (sample_mean-population_mean) / (populationSigma /math.sqrt(sample_count))
print(f'Z score value is: {zScore}')

Z score value is: 3.2


If we chek this value in z table, then we get 0.9993  <br>
means probability of less than 108 is 99.93 and more than or equal to 108 is 0.007. It is less than 0.05. soo we will reject
null hypothesis

## Problem Statement 2:
In one state, 52% of the voters are Republicans, and 48% are Democrats. In a second
state, 47% of the voters are Republicans, and 53% are Democrats. Suppose a simple
random sample of 100 voters are surveyed from each state.
What is the probability that the survey will show a greater percentage of Republican
voters in the second state than in the first state?

In [35]:
n1 = 100  # Voters sampled from first state
n2 = 100  # # Voters sampled from second state

P1 = 0.52
P2 = 0.47

Q1 = 0.48 
# (1-P1, this is proportion of non republican voters in first state)

Q2 = 0.53
# (1-P2, this is proportion of non republican voters in first state)

mu = P1 - P2

std = math.sqrt(((P1*Q1) / n1) + ((P2*Q2) / n2))

In [36]:
print(f'The value of mu: {mu} and the value of std is: {std}')

The value of mu: 0.050000000000000044 and the value of std is: 0.07061869440877536


In [37]:
x = 0
zScore = (x-mu) / std

In [38]:
print(f'Z score value is: {zScore}')

Z score value is: -0.7080278164104213


The probability that the survey will show greater percentage of republican voters in the second state than in the first state. 

## Problem Statement 3:
You take the SAT and score 1100. The mean score for the SAT is 1026 and the standard
deviation is 209. How well did you score on the test compared to the average test taker?

In [39]:
# lets
x = 1100  # score 
mu = 1026   #population mean
standard_dev = 209

In [40]:
zScore = (x - mu)/standard_dev

In [41]:
print(f'Z score value is: {zScore}')

Z score value is: 0.35406698564593303


here we can conclude that the x value is 0.35 standard deviation above the mean.